src/output/html.js

```diff
@@ -453,9 +453,9 @@ const getHtmlLinks = () => {
   }, {});
   if (leagueRef.includeOverall) {
     if (leagueRef.hasTeams) {
-      addLinks(links, getLocalization().overall, "team");
+      addLinks(links, "overall", "team");
```

Comment:
I made a mistake adding this change. Have now added a bunch of extra e2e tests that now test overall and the results html, so hopefully mistakes like this don't creep in in the future :D - I fixed it by passing in a displayName to the addLinks method. Does mean that the name of the html file contains the word 'overall' but as that's more of a key to the file I'm ok with that.